src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -113,6 +121,33 @@ export class SkyLookupComponent
   @Input()
   public idProperty: string;
 
+  /**
+   * Specifies the selection mode that determines whether users can select one item or
+   * multiple items. The valid options are `single` and `multiple`.
+   * @default "mulitple"
```

Comment:
```suggestion
   * @default multiple
```
I don't think we need quote marks around this default value. Steve told me to use them for `string` properties, but it looks like this is an `enum`. ... Feel free to add them back if we do need them, but we still need to fix the spelling here.

Also, just out of curiosity, why didn't we just implement this property as a boolean that toggled between single- and multi-select? Are we expecting to need more options at some point?

Reply:
I think the quotes are appropriate here as the enum is a `string` enum. And we appear to have precedence for leaving the quotes on `string` enums in both popovers and dropdowns.

I don't know that we are expecting more options. However, I like the enum because it is easier to follow what is going on throughout the code and is clear which options is on. I also like that it gives us the opportunity to add more options if we find a reason to do so in the future.

Reply:
Sounds good. I wasn't sure if the enum was a `string`, so I wanted to raise the question. Same with the question about boolean. It just occurred to me, so I figured it was worth asking. Thanks!